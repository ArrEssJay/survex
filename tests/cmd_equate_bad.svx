; pos=fail warn=0 error=3
; We used to report "End of line not blank" but 1.2.22 gives "Character "=" not
; allowed in station name (use *SET NAMES to set allowed characters)" instead.
*fix 1 reference 0 0 0
*equate 1 =
*equate 1 2 =
*equate 3 4 
*equate 2 1 3 ; test
*equate foo
