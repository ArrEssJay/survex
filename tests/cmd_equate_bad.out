./cmd_equate_bad.svx:5:11: error: Character "=" not allowed in station name (use *SET NAMES to set allowed characters)
./cmd_equate_bad.svx:6:13: error: Character "=" not allowed in station name (use *SET NAMES to set allowed characters)

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 4 survey stations, joined by 3 legs.
There is 1 loop.
Survey has 2 connected components.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   2 1-nodes.
   2 2-nodes.
There were 0 warning(s) and 2 error(s) - no output files produced.
